debezium-core/src/main/java/io/debezium/relational/mapping/MaskStrings.java

```diff
@@ -35,6 +46,24 @@ public MaskStrings(String maskValue) {
         this.converter = new MaskingValueConverter(maskValue);
     }
 
+    /**
+     * Create a {@link ColumnMapper} that masks string values by hashing the input value.
+     *
+     * @param salt the salt that is used within the hash function
+     * @param hashAlgorithm the hash function that is used to mask the columns string values written in source records;
+     *                      must be on of Java Cryptography Architecture Standard Algorithm {@link MessageDigest}.
+     * @throws IllegalArgumentException if the {@param salt} or {@param hashAlgorithm} are null
+     */
+    public MaskStrings(byte[] salt, String hashAlgorithm) {
+        if (salt == null) {
```

Comment:
I'm going to change it.

`== null` is used a lot in the code. Could it be a separate PR to use `Objects#requireNonNull()` everywhere? WDYT?

Reply:
I would say it would make sense to use `Objects#requireNonNull()` in the changes pertinent to this PR and then we can follow-up with other uses of `== null` in a follow-up PR as to keep the most relevant code changes together.

Reply:
Yes, let's keep general "clean-up" separate from the issue at hand. Personally I don't see a strong need to change this across the board, but if feel like doing it, a PR to do so will surely be welcomed.